src/main/java/uk/gov/hmcts/ccd/data/SecurityUtils.java

```diff
@@ -54,7 +56,13 @@ public HttpHeaders userAuthorizationHeaders() {
     }
 
     public UserInfo getUserInfo() {
-        return idamRepository.getUserInfo(getUserToken());
+        UserInfo userInfo = idamRepository.getUserInfo(getUserToken());
+        if (userInfo != null) {
+            log.info("SecurityUtils retrieved user info from idamRepository. User Id={}. Roles={}.",
```

Comment:
we are already logging in idamRepository, it is not needed again here... it will log the same info multiple times